core/sdk/src/main/java/com/mongodb/stitch/core/internal/net/Transport.java

```diff
@@ -19,6 +19,8 @@
 import java.io.IOException;
 
 public interface Transport {
+  int MAX_REQUEST_SIZE = 1 << 22;
```

Comment:
[nit] `1 << 22` probably doesn't mean much to most people. Either use `4194304` and/or leave a comment explaining what's going on here.